gapic/templates/examples/feature_fragments.j2

```diff
@@ -25,44 +25,78 @@ There is a little, but not enough for it to be important because
 
 {# response handling macros #}
 
+{% macro licenseSection(fileHeader, sample, callingForm) %}
```

Comment:
Please use `snake_case` macro names and variable names (see PEP 8).

Reply:
Given that that ship has sailed in the other parts of the fragments template, would you be okay with a separate, subsequent PR that snakeifies all the macro names?

Reply:
Tracked by #144

Reply:
Yes.